files/default/csr.py

```diff
@@ -116,6 +116,11 @@ def store(self):
             with open(join(cert_dir, self._config.certificate_file), "wt") as fd:
                 fd.write(self._certificate)
 
+        if self._certificate is not None and self._intermediate_ca is not None:
+            with open(join(cert_dir, "elastic_host.pem"), "wt") as target:
```

Comment:
I have an attribute for elastic_host.pem file, but didn't add it to the config.ini since i wasn't using it there. https://github.com/logicalclocks/kagent-chef/blob/master/attributes/default.rb#L101
We should use it here, instead of explicitly writing "elastic_host.pem"

Reply:
Done